### PR TITLE
CNF-11561: fix non-vendor snyk errors

### PR DESCRIPTION
**Pull request by pixelsoccupied**
Fix remaining errors from https://github.com/openshift/release/pull/49688

The dot file was regenerated with the following
```
snyk ignore --file-path='vendor/**' --file-path-group='global'   
snyk ignore --file-path='**/*_test.go' --file-path-group='global'
```

### CODE REVIEW

**Comment by pixelsoccupied** [comment]
/unhold
all issues should be fixed now

File: internal/backuprestore/restore.go [hunk 1]
```diff
@@ -232,7 +232,7 @@ func (h *BRHandler) LoadRestoresFromOadpRestorePath() ([][]*velerov1.Restore, er
 
 		// The returned list of entries are sorted by name alphabetically
 		restoreDirPath := filepath.Join(OadpRestorePath, restoreSubDir.Name())
-		restoreYamls, err := os.ReadDir(filepath.Join(hostPath, restoreDirPath))
+		restoreYamls, err := os.ReadDir(filepath.Clean(filepath.Join(hostPath, restoreDirPath)))
```
Comment (donpenney) [inline internal/backuprestore/restore.go:235]: Too bad this isn't reported by golangci-lint / gosec.

**Review by donpenney — APPROVED** [review]

File: internal/backuprestore/restore.go (hunk 1 above)
Comment (pixelsoccupied) [reply internal/backuprestore/restore.go:235]: Yeah pretty strange

**Pull request merged**